### DISCUSSION TITLE
PolarFire SOC and SoftConsole "Trace" Feature

### DISCUSSION THREAD

**Discussion by haroldnelson-2021** [polarfire-soc/polarfire-soc-documentation#487] [General]
Hello, the MSS Technical Reference Manual describes the PF SOC's Trace feature and usage in some detail.  It references the SoftConsole documentation for seeing more information, however the SoftConsole documentation indicates this is a TBD feature or not yet implemented.  Please clarify.

**Comment by sanjeevb-bansal** [comment]
Hi Harold, 

Following our internal discussions, we want to clarify that if our customer intends to utilize the Trace feature of the PolarFire SoC, as detailed in the MSS TRM, they will need to use Lauterbach's TRACE32® tool, available at https://www.lauterbach.com/. SoftConsole does not (and will not) support collecting or parsing Trace data from the PolarFire SoC device in a user-friendly GUI, as provided by Lauterbach's TRACE32. We will be raising an internal ticket to update the SoftConsole documentation accordingly.

Thanks
Sanjeev
